.github/workflows/default.yml

```diff
@@ -3,10 +3,15 @@ on:
 
 jobs:
   test:
+    strategy:
+      matrix:
+        ruby: [2.3, 2.6, 2.7]
```

Comment:
Do you mean 2.3 here? I'd imagine we want to change the gemspec to be 2.5 and start there?

Reply:
Indeed I do. Since this is [the minimum version supported by the gem](https://github.com/alphagov/govuk-connect/blob/master/govuk-connect.gemspec#L10), we should be testing against it. This [will go up to 2.4 in one of my other PRs](https://github.com/alphagov/govuk-connect/pull/11/files#diff-b5f414ac5df0178953e7c003916546f3R10). I didn't want to change it here, as there wasn't any motivation to do so.

Does raise the question: which Ruby versions _do_ we support?

Reply:
For gems we expect to only be internal usage I'd suggest 2.6 and up since we've updated all our infrastructure to be 2.6 and we'd want it to work against current version too.

For gems that we'd be happy with external parties using 2.5 seems the reasonable minimum since that is the lowest non eol-version of Ruby. (we'll pretend we don't still use Ruby 1.9 with Puppet 😬)

I'd go straight to 2.5 or 2.6 in your other PR and skip 2.4.

Reply:
Sure, I've updated to reflect that.

(Weirdly, I read somewhere 2.4 was still supported.)